commands/team_e2e_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+)
+
+func (s *MmctlE2ETestSuite) TestRenameTeamCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForAllClients("Error renaming team which does not exist", func(c client.Client) {
+		printer.Clean()
+		oldTeamName := "existingName"
+		cmd := &cobra.Command{}
+		args := []string{""}
+		args[0] = oldTeamName
+		cmd.Flags().String("display_name", "newDisplayName", "Team Display Name")
+
+		err := renameTeamCmdF(c, cmd, args)
+		s.Require().EqualError(err, "Unable to find team 'existingName', to see the all teams try 'team list' command")
+	})
+
+	s.RunForSystemAdminAndLocal("Rename a valid team", func(c client.Client) {
+		printer.Clean()
+
+		cmd := &cobra.Command{}
+		args := []string{""}
+		args[0] = s.th.BasicTeam.Name
```

Comment:
Same here, no real need for two assignments.

Reply:
Done.